frontend/src/pages/projects/const.ts

```diff
@@ -1 +1 @@
-export const DEFAULT_PVC_SIZE = 20;
+export const DEFAULT_PVC_SIZE = 10;
```

Comment:
I don't think we need this constant, can we not just get the pvc size out of the OdhDashboardConfig?

Reply:
Ah I see what is happening -- take this constant to the file -- no one should use this, everyone should use your hook. Maybe also keep it at 20? 🤔